fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -275,12 +275,14 @@ public Response doGet(String path, String mimeType, String provider, String outc
 
     private void checkExportStatus(boolean isCheckPatient, boolean s3, List<String> types) throws Exception {
         Response response;
+        // We're doing a tight loop here so we can do an immediate DELETE it.
+        ZonedDateTime startTime = ZonedDateTime.now();
         do {
             response = doGet(exportStatusUrl, FHIRMediaType.APPLICATION_FHIR_JSON, "default", "default");
             // 202 accept means the request is still under processing
             // 200 mean export is finished
             assertEquals(Status.Family.familyOf(response.getStatus()), Status.Family.SUCCESSFUL);
-            Thread.sleep(1000);
+            waitForSetPeriodOfTime(startTime, 5, 1000);
```

Comment:
I looked this one up and the second argument is the maximum time to wait in minutes.  So essentially this is saying throw and exception to break us out of this loop if its been five minutes.  Smart.